lib/whitehall_importer.rb

```diff
@@ -10,7 +10,7 @@ def self.import(whitehall_document)
     )
 
     begin
-      Import.call(whitehall_document)
+      Import.call(whitehall_document, record)
```

Comment:
As the record has access to the Whitehall document we may be able to just push that in.